packages/gatsby-theme-store/src/sdk/orderForm/LazyProvider.tsx

```diff
@@ -1,13 +1,21 @@
 import type { PropsWithChildren } from 'react'
 import React, { lazy, Suspense } from 'react'
 
-const LazyProvider = lazy(() =>
-  import('./Provider').then(({ Provider }) => ({
-    default: Provider,
-  }))
-)
+const isOfflinePage = window.location.pathname.startsWith('/offline')
```

Comment:
why do we need this?

Reply:
Because the lazy import would not work offline, and since this component is wrapping every page, including the offline one, it would break

Reply:
lol. @igorbrasileiro , that's why we can't use this approach for lazy loading providers then. Ok

Reply:
Maybe we can handle the promise failing instead.

```js
const noop = ({ children }) => children

const LazyProvider = lazy(() => 
  import('./Provider').then(({ Provider }) => ({
    default: Provider,
  }).catch(() => { return noop })
)
```